airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/bump_version/pipeline.py

```diff
@@ -153,9 +153,10 @@ def __init__(
 
     async def get_repo_dir(self) -> Directory:
         if not self.repo_dir:
-            repo_dir = await self.context.get_repo_dir()
-            self.repo_dir = repo_dir
-        return repo_dir
+            self.repo_dir = await self.context.get_repo_dir()
```

Comment:
do you know why `self.context.get_repo_dir()` can return None?

Reply:
It doesn't, which is weird. Mypy type checking is aggressive about None-checking self.repo_dir because it can be none coming in. 

However this gave me a new idea to fix it (checking `if self.repo_dir is None` instead of `if not self.repo_dir`, which seems to appease mypy, even though the latter should cover the None case 🙃 fixed